frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.unit.spec.tsx

```diff
@@ -77,27 +87,60 @@ const mockDataset = new Question(
   metadata,
 );
 
-const databaseWithEnabledActions = createMockDatabase({
-  settings: { "database-enable-actions": true },
-});
+const mockDatasetWithClauses = new Question(
```

Comment:
Can we get a Question instance via `metabase.question(card.id)` and creating Questions directly? Ideally creating Questions should be forbidden but we don't have ESLint rules for that.

Reply:
I updated the entire file: 0913af1583d5d4cf81b78c1f5cb5871e91cc606f